src/request_collector/cli.py

```diff
@@ -109,7 +109,7 @@ def main(
         monitoring_channel,
     )
 
-    database = StateDBSqlite(state_db)
+    database = SharedDatabase(state_db)
```

Comment:
What happens if the DB doesn't exist yet at this point?

Reply:
We get an error about missing tables. I'll add a more informative error message.

Reply:
We now get
```
sqlite3.OperationalError: unable to open database file
```
right after the log message `Opening database at ...`. That's not super explanatory, but ok. We can add more detail once we have a clear idea how MR and RC are usually started.